std_msgs/Int8 user_cmd
std_msgs/Int8 cmd_eo_mode
std_msgs/Float64 move_pan
std_msgs/Float64 move_tilt
std_msgs/Float64 eo_zoom
